udata/harvest/tests/test_base_backend.py

```diff
@@ -281,6 +287,78 @@ def test_harvest_datasets_get_deleted(self):
         for item in job.items:
             assert item.dataset is None
 
+    def test_no_datasets_duplication(self, app):
+        limit = app.config["HARVEST_AUTOARCHIVE_GRACE_DAYS"]
+        last_update = datetime.utcnow() - timedelta(days=limit - 1)
+        reuse_remote_id_uri = "http://example.com/reuse-this-uri"
+        nb_datasets = 3
+        source = HarvestSourceFactory(
+            config={
+                "nb_datasets": nb_datasets,
+                "dataset_remote_ids": [reuse_remote_id_uri],
```

Comment:
Don't we want to have either `nb_datasets` OR `dataset_remote_ids`?
Else I find it hard to follow what happens when we have both :)

Reply:
I updated the tests: https://github.com/opendatateam/udata/pull/3219/commits/aec0a6a5bf36e5a866bb7b4b6c73f50db47786aa
What do you think?

Reply:
Yes, I find it better, thank you!